### PR TITLE
OSDOCS-12181:Added FIPS cryptography option in create classic cluster  ROSA CLI workflow.

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-12181

Link to docs preview:
[Creating a cluster with customizations using the CLI](https://82690--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-sts-creating-cluster-customizations-cli_rosa-sts-creating-a-cluster-with-customizations)

Peer review:
- [x] Peer reviewer has approved this change.

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 1]
```diff
@@ -308,14 +311,15 @@ PVs created by using any other storage class are still encrypted, but the PVs ar
...
+<13> Optional: Enable this option if you require your cluster to be FIPS validated. Selecting this option means the encrypt etcd data option is enabled by default and cannot be disabled. You can encrypt etcd data without enabling FIPS support.
+<14> Optional: Enable this option only if your use case requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default. With this option, the etcd key values are encrypted but not the keys.
 +
 [IMPORTANT]
 ====
 By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Red{nbsp}Hat recommends that you enable etcd encryption only if you specifically require it for your use case.
 ====
 +
-<14> The output includes a custom command that you can run to create a cluster with the same configuration in the future.
+<15> The output includes a custom command that you can run to create a cluster with the same configuration in the future.
```
Comment (dfitzmau) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:322]: Is "in the future" necessary here? Running the custom command is the future action and this has been stated.

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 2]
```diff
@@ -308,14 +311,15 @@ PVs created by using any other storage class are still encrypted, but the PVs ar
 ====
 
 <12> Optional: You can select additional custom security groups to use in your cluster. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups for the default machine pools after you create the machine pool. For more information, see the requirements for _Security groups_ under _Additional resources_.
-<13> Optional: Enable this option only if your use case requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default. With this option, the etcd key values are encrypted but not the keys.
+<13> Optional: Enable this option if you require your cluster to be FIPS validated. Selecting this option means the encrypt etcd data option is enabled by default and cannot be disabled. You can encrypt etcd data without enabling FIPS support.
+<14> Optional: Enable this option only if your use case requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default. With this option, the etcd key values are encrypted but not the keys.
```
Comment (dfitzmau) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:315]:
```suggestion
<14> Optional: Enable this option if your use case only requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default. With this option, the etcd key values are encrypted but not the keys.
```

**Review by dfitzmau — COMMENTED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc (hunk 1 above)
Comment (AedinC) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:322]: Yeah I get what you're saying. I don't have a problem with it but could change it to ´The output includes a custom command that you can run to create another cluster with the same configuration.'

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**